test/plugins/dereferenceUses.07.svg

```diff
@@ -0,0 +1,476 @@
+Check whether plugin can handle more complex SVGs with attribute override between <use> elements and their referenced elements.
+
+===
```

Comment:
Do we need such big test? I see a lot of exactly the same cases.

Reply:
No, the test is redundant. I removed it.